frozendict/__init__.py

```diff
@@ -1,8 +1,12 @@
-import collections
 import operator
 import functools
 import sys
 
+try:
+    # Python 3.3+
+    from collections import Mapping
```

Comment:
Shouldn't it be the other way? This would mean collections is always looked up first emitting deprecation warning in versions less than 3.9 .

Reply:
Yes, that was my intention, to match the comment.

Thanks for catching it, updated!